packages/grafbase-sdk/src/grafbase-schema.ts

```diff
@@ -60,7 +60,9 @@ export interface IntrospectParams {
   namespace?: boolean
 }
 
-export class SingleGraph {
+const FEDERATION_VERSION = '2.3'
+
+export class Graph {
```

Comment:
shouldn't this be StandaloneGraph?

Reply:
We can change it, internally not sure it matters since graph is pretty descriptive

Reply:
It might be good since the other variant also has the word graph and isn't an extension but we could iterate on it later